finagle-mysql/src/main/scala/com/twitter/finagle/mysql/Parameter.scala

```diff
@@ -0,0 +1,37 @@
+package com.twitter.finagle.exp.mysql
+
+import com.twitter.finagle.exp.mysql.transport.BufferWriter
+import language.implicitConversions
+
+trait Parameter {
+  type A
```

Comment:
@mosesn It wouldn't make sense. It's supposed to be used as an existential type.